projects/isy-angular-widgets-demo/src/app/feature/objekt-anzeigen/objekt-anzeigen.component.html

```diff
@@ -142,22 +141,24 @@
           </div>
         </div>
         <div class="col-12 md:col-6 xl:col-4">
-          <div class="field">
-            <span class="p-float-label">
-              <p-calendar
-                inputId="birth-date"
-                dateFormat="dd.mm.yy"
-                formControlName="birthDate"
-                [showIcon]="true"
-                styleClass="w-full"
-                appendTo="body"
-              >
-              </p-calendar>
-              <label for="birth-date">
-                {{ 'isyAngularWidgetsDemo.labels.geburtsdatum' | translate }}
-              </label>
-            </span>
-          </div>
+          <!-- Form-Wrapper-Komponente, die ein Label, welches automatisch zum Eingabefeld zugeordnet wird, 
+          ein beliebiges PrimeNG Eingabefeld und eine Validierungsnachricht beinhaltet.
+          Die Formcontrol nutzt einen benutzerdefinierten Validator. -->
+          <isy-form-wrapper
+            label="{{ 'isyAngularWidgetsDemo.labels.geburtsdatum' | translate }}"
+            fieldId="birth-date"
+            [control]="personalInfoForm.controls.birthDate | formControl"
+            [validationMessages]="{PAST: 'isyAngularWidgetsDemo.messages.dateInPastError' | translate}"
```

Comment:
Should be lower case
```suggestion
            [validationMessages]="{past: 'isyAngularWidgetsDemo.messages.dateInPastError' | translate}"
```